dashboard/src/components/status/Status.vue

```diff
@@ -0,0 +1,44 @@
+<!--
+  - Copyright (C) 2016-2020 the original author or authors
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as published
+  - by the Free Software Foundation, either version 3 of the License, or
+  - (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  -->
+
+<template>
+	<div class="Status">
+		I am inevitable
+
+		<div class="shard">
+			<p>{{ shard.id }}</p>
+			<p>{{ shard.status }}</p>
```

Comment:
For some reason shard id and status return undefined.
I'll need some help to figure out why.